spec/features/user_deletes_account_spec.rb

```diff
@@ -1,17 +1,57 @@
 require 'rails_helper'
 feature "User deletes account" do
+  let!(:user) { FactoryGirl.create(:user) }
+
   scenario 'user navigates to edit account page' do
     user_login
     click_on 'Edit registration'
 
     expect(page).to have_content('Cancel my account')
   end
   scenario 'user deletes account' do
-    user_login
+    user1 = User.create(
+      email: "tim@odsc.com",
+      password: "11223344"
+    )
+
+    Profile.create(
+      username: "Xander321",
+      location: "Boston",
+      avatar_url: "https://s-media-cache-ak0.pinimg.com/236x/50/bb/11/50bb1149e480e46d721d8a813a8ef3d4.jpg",
+      pun: "Isn't this punny?",
+      user: user1
+    )
+
+    pyramid = Landmark.create(
+      name: "Pyramids of Giza",
+      location: "Egypt",
+      image: "https://upload.wikimedia.org/wikipedia/commons/e/e3/Kheops-Pyramid.jpg",
+      description: "hot, but awesome.",
+      user: user1
+    )
+
+    Review.create(
+      body: "This place was kinda dope",
+      landmark: pyramid,
+      user: user1,
+      votes: "4"
+    )
```

Comment:
Why weren't factories used here? And why was a new user created instead of using the one provided by the factory at the top of the test?